JSTask5.js

```diff
@@ -0,0 +1,26 @@
+// Write a javascript function code to dynamically grab object keys and use the keys as part of a method.
+// Sample :
+// const data = [ { title: "title1", subtitle:"sub1"}, { title: "title2", subtitle:"sub2"}, { title: "title3", subtitle:"sub3"} ];
+// searchkey = 'title2';
+// Expected Output : [ {
+// "title": "title2",
+// "subtitle": "sub2", }
+// ]
+
+const data =[
+    {title: "title1", subtitle: "sub1"},
+    {title: "title2", subtitle: "sub2"},
+    {title: "title3", subtitle: "sub3"}
+];
+
+let search = "title2";
+
+let filterdata = data.filter(
```

Comment:
Applicable only for key "title". The code should handle searching any key in the object i.e., if I give "sub1" as searchKey, it should return me 
[
{
 title: "title2",
subtitle: "sub2"
}
]